src/components/MiscellaneousPanel.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import useCPU from '../hooks/use-cpu';
+import Loader from './Loader';
+import MemoryGraph from './MemoryGraph';
+
+// Honestly, we might think about getting rid of this component
+// its just a placeholder to store the graph, and potentially endian feature
+const MiscellaneousPanel = () => {
```

Comment:
why is it called miscellaneous panel? i thought it was for showing memory utilization?

Reply:
im going to remove this component and just put the memory component instead